CHANGELOG.md

```diff
@@ -2,6 +2,49 @@
 
 The changelog for `MessageKit`. Also see the [releases](https://github.com/MessageKit/MessageKit/releases) on GitHub.
 
+## Upcoming release
```

Comment:
Can we omit lines 5-15 from this PR? We can have an `## Upcoming release` section on the /development branch, but I don't think it makes sense to have it on /master. It is the first thing a framework users sees, and as-is it looks like a forgotten placeholder.